test/battle/trainer_control.party

```diff
@@ -5,6 +5,7 @@ Pic: Red
 Gender: Male
 Music: Male
 Double Battle: No
+Difficulty: Normal
```

Comment:
Does this need to be here?
Doesn't it default to Normal and there is no tests written for difficulty handling.

Reply:
https://github.com/rh-hideout/pokeemerald-expansion/pull/5337#issuecomment-2402280930

I was instructed to add it. I don't feel strongly either way.

Reply:
Ah, doesn't matter then.